Engine/Graphics/GL2.cpp

```diff
@@ -31,11 +31,7 @@ int GPUController::getImageFormatGL2(GPU_Image *image) {
 }
 
 void GPUController::printBlitBufferStateGL2() {
-	auto cdata = static_cast<GPU_CONTEXT_DATA *>(GPU_GetCurrentRenderer()->current_context_target->context->data);
-	if (cdata->blit_buffer_num_vertices > 0 && cdata->last_target && cdata->last_image)
-		sendToLog(LogLevel::Info, "Blit buffer size: %u\n", cdata->blit_buffer_num_vertices);
-	else
-		sendToLog(LogLevel::Info, "Blit buffer empty.\n");
+	sendToLog(LogLevel::Warn, "This is deprecated!");
```

Comment:
Why is this deprecated?

Reply:
It doesn't compile with those for some undefined reference reason, I cannot find any part of the script that uses it so I just removed it

Reply:
Eh, it is useful for debugging. Did you change SDL_gpu?

Reply:
at the time yes but it should be revertible now